sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -178,6 +177,8 @@ export interface ManagementClientOptions {
   audience?: string;
 }
 
+type SendManagementRequestOptions2 = SendManagementRequestOptions & OperationOptionsBase;
```

Comment:
Can't we just add whatever fields we need to the original SendManagementRequestOptions? Having a '2' just appended seems odd to me.

Reply:
Just a quick draft, I'll change that :)